src/SelectBox/SelectBox.js

```diff
@@ -0,0 +1,79 @@
+import React, { useRef, useEffect } from 'react';
+import propTypes from 'prop-types';
+import {
+  StyledOptionsList,
+  StyledOptionsListItemInnerButton,
+  StyledOptionsListItem
+} from './SelectBox.styles';
+import { StyledCutout } from '../Cutout/Cutout';
+
+const SelectBox = React.forwardRef(function SelectBox(props) {
+  const { options, value, onSelect, width, height } = props;
```

Comment:
What do you think about destructuring those props on the component declaration?

Like:
```
const SelectBox = React.forwardRef(function SelectBox({
  options,
  value,
  onSelect,
  width,
  height
}) {
  // stuff
});
```

Reply:
Sure, i will make those changes! I copy pasted "Select" component as a base for this component, that's why i missed the eslint disable comment and destructured props that way hahah thanks for pointing out!

Reply:
Actually let's keep it how it is in other components to keep it consistent